canopen/lss.py

```diff
@@ -197,19 +198,22 @@ def activate_bit_timing(self, switch_delay_ms):
         message = bytearray(8)
 
         message[0] = CS_ACTIVATE_BIT_TIMING
-        message[1:3] = struct.pack('<H', switch_delay_ms)
+        message[1:3] = struct.pack("<H", switch_delay_ms)
```

Comment:
We shouldn't get started with this, it has potential for endless discussions and no real benefit.  I actually configure Black with `skip-string-normalization = true` locally, to avoid such noise.